cmd/ethrex/utils.rs

```diff
@@ -82,9 +82,20 @@ pub fn read_block_file(block_file_path: &str) -> Block {
 }
 
 pub fn read_genesis_file(genesis_file_path: &str) -> Genesis {
-    let genesis_file = std::fs::File::open(genesis_file_path).expect("Failed to open genesis file");
+    let path = get_genesis_path(genesis_file_path);
+    let genesis_file = std::fs::File::open(path).expect("Failed to open genesis file");
     decode::genesis_file(genesis_file).expect("Failed to decode genesis file")
 }
+// If genesis_file_path is one of the public networks return the path to its genesis.json file, if not return the inputed path
+fn get_genesis_path(genesis_file_path: &str) -> &str {
```

Comment:
nit: argument you probably be `network` to make it clear it is not necesary a path. Also I would not call this function from inside `read_genesis_file `. Instead I would do something like:

let genesis_path = get_genesis_path(network);
let genesis = read_genesis_file(genesis_path);

Also, Instead of `genesis_file_path ` receiving a string, it should receive a `Path`.

Also,  maybe `network` cli param receive a custom type, that is like an enum that can be a PublicNetwork, or (Genesis) Path.

Reply:
I think this would be great. nit: instead of path `GenesisPath(Path)`